service-orchestration/README.md

```diff
@@ -567,9 +567,9 @@ Running executable
 
 Let's test the use case by sending a request to the service.
 
-#### Start the back end service
+#### Start the backend service
 
-Download the JAR file for the backend service from [here](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/) and execute the following command to start the service:
+Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar) and execute the following command to start the service:
 
 ```bash
```

Comment:
Let's indent this

Reply:
This part doesn't belong to a bullet or numbering. This is the content of the header. I feel like we don't need to intend this

Reply:
Yes my bad